### ISSUE TITLE
Enhancement: Skip on curtain click option.

### ISSUE THREAD

**Issue by hugo-azevedo** [abhikmitra/ng-joyride#40]
Or is there a way to do this already?

Thanks!

**Comment by abhikmitra** [comment]
You want the user to skip on click of the curtain ? Umm, the way I see it ,its not a super important feature right.

**Comment by hugo-azevedo** [comment]
Fair enough. It's a "would-be-nice-to-have", which would make joyride behave like curtains work in other environments. 

It would allow the user to continue using the website, if a joyride step were to fail, because, when it does, the only thing left is the curtain, which doesn't allow the user to click on anything else.

Thanks anyway.
